### PR TITLE
name change

### PR DESCRIPTION
**Pull request by JowieXiang**

### CODE REVIEW

**Review by janbehrens — APPROVED** [review]

File: webapp/scripts/messages.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "1": {
             "message_id": "add_location.1",
             "message": {
-                "text": "A location has already been set. If you want to set a new location, the existing location will be overwritten. Do you want to set a new location?"
+                "text": "The basemaps have already been set. If you want to set a new set of basemaps, the existing basemaps will be overwritten. Do you want to set new basemaps?"
```
Comment (janbehrens) [inline webapp/scripts/messages.json:6]: Why plural? There can be only one basemap.

Comment (JowieXiang) [reply webapp/scripts/messages.json:6]: the base map contains three layers: waterline, roads and buildings. So I suppose they are three maps? Or should they be called collectively the 'basemap'?

Comment (janbehrens) [reply webapp/scripts/messages.json:6]: As I understood it, the _basemap_ is the loaded OSM file (previously the "location"), and the roads, ater lines etc. are the _base layers_... but I see that your view is a valid one, too. In that case, please name the layer group consistently --> https://github.com/citysciencelab/open-city-toolkit/blob/master/webapp/views/map/script.js#L131

Comment (JowieXiang) [reply webapp/scripts/messages.json]: Your statement is more valid. So I changed accordingly.

**Pull request merged**